lib/util/escapeRegExp.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Escapes RegExp special characters from a string
```

Comment:
I understand what you were trying to do here, but I'm explicitly saying that regular expression patterns *never belong in an eslint config*. This is also the position of the eslint core team, fwiw.